CHANGELOG.md

```diff
@@ -38,7 +38,7 @@
   * Implement `TryFrom<CasmContractClass> for Program`
   * Add `Cairo1HintProcessor`
 
-* Add `CairoRunner::get_program method` [#1123](https://github.com/lambdaclass/cairo-rs/pull/1123):
```

Comment:
Why was this entry removed?

Reply:
it was duplicated :P 
If you check down the changelog is added in the 0.4.0 version

Reply:
Nice!